Users/Docker_Image.rst

```diff
@@ -16,6 +16,7 @@ system, Docker allows applications to use the same kernel as the system
 that they're running on and only requires applications being shipped
 with things not already running on the host computer. This gives a
 significant performance boost and reduces the size of the application.
+Also, Docker does not support Python 2 analysis.
```

Comment:
I think this belongs to "coala as a docker image". Docker itself supports every kind of software more or less ;)

Reply:
You could put this into a note, like
```
.. note::

The coala Docker image does not support Python 2 analysis.
```
Or so